pupil_src/shared_modules/calibration_routines/calibrate.py

```diff
@@ -46,13 +46,13 @@ def calibrate_2d_polynomial(cal_pt_cloud,screen_size=(1,1),threshold = 35, binoc
         logger.info('used {} data points out of the full dataset {}: subset is {:.2f} percent'.format(
             used_num, complete_num, 100*float(used_num)/complete_num))
 
-        return map_fn, err_dist <= threshold, (cx, cy, model_n)
+        return map_fn, err_dist <= threshold, ([p.tolist() for p in cx], [p.tolist() for p in cy], model_n)
```

Comment:
Exactly. Lists of ndarrays. Alternatively, one could use `np.asrray(obj).tolist()`.